test/e2e/__init__.py

```diff
@@ -24,6 +26,7 @@
 CONFIG_RESOURCE_PLURAL = "endpointconfigs"
 MODEL_RESOURCE_PLURAL = "models"
 ENDPOINT_RESOURCE_PLURAL = "endpoints"
+DATA_QUALITY_JOB_DEFINITION_RESOURCE_PLURAL = "dataqualityjobdefinitions"
```

Comment:
why are these defined here vs resource test files ?
hpo/processing are still doing it differently. We will need to review test PRs carefully for a while to catch these diffs.

Reply:
resources that are being used by multiple tests have plurals defined here in a common location